tools/report-converter/tests/unit/tidy_output_test_files/tidy3_hh.plist

```diff
@@ -4,64 +4,6 @@
 <dict>
```

Comment:
how can we generate a report only for a header file? shouldn't be a c/cpp file be on the path? there is no hash in here how will we store this, as a separate report, I think this should be part of the tidy3_cpp.plist file

Reply:
Check the following example: https://github.com/csordasmarton/codechecker/blob/report_converter_support_cppcheck/tools/report-converter/tests/unit/tidy_output_test_files/tidy3.out

If a bug can be found in a header file we do not know more information about this so we will put the report to a plist file which refers to a header file.
```sh
files/test3.hh:6:6: warning: Dereference of null pointer (loaded from variable 'x') [clang-analyzer-core.NullDereference]
  *x = 42;
     ^
```

Reply:
This whole section should be in one report file, this is one analyzer report with multiple steps which should not be split to multiple plist files.
```
files/test3.hh:6:6: warning: Dereference of null pointer (loaded from variable 'x') [clang-analyzer-core.NullDereference]
  *x = 42;
     ^
files/test3.cpp:4:3: note: 'x' initialized to a null pointer value
  int* x = 0;
  ^
files/test3.cpp:6:11: note: Assuming 'argc' is > 3
  if (foo(argc > 3)) {
          ^
files/test3.cpp:6:3: note: Taking true branch
  if (foo(argc > 3)) {
  ^
files/test3.cpp:7:9: note: Passing null pointer value via 1st parameter 'x'
    bar(x);
        ^
files/test3.cpp:7:5: note: Calling 'bar'
    bar(x);
    ^
files/test3.hh:6:6: note: Dereference of null pointer (loaded from variable 'x')
  *x = 42;
     ^
```

and I do not see the checker name `clang-analyzer-core.NullDereference` in the created plist file.